### DISCUSSION TITLE
Novice - Cannot Update Files - Commit failed

### DISCUSSION THREAD

**Discussion by Quentin-M58** [community/community#111229] [Repositories]
### Select Topic Area

Bug

### Body

Hi All

First of all thank you in advance for any assistance.
I am a novice when it comes to using github.

I have a problem since today with github.com .
i have a problem with an upload of files because i have this message : Committing the files to the repository failed . 

So if I can be assisted with what I need to do to get back to point when I can edit and post changes as I was able to previously .
I'm sorry if my speaking is not good
Many Thanks

**Comment by LORDARRAS3000TESTER** [comment]
Same

**Comment by mmcomputer** [comment]
Same problem here

**Comment by OsamaMohamedAtta** [comment]
i have same problem

**Comment by xxxtianxxx** [comment]
same experience, do you already solve the issue?

**Comment by Humphydumphy1** [comment]
I also have this issue committing the files to the repository. you should double-check ur permissions and resolve any conflicts before trying to commit again.. Hope it helps broski :))

**Reply by Quentin-M58** [reply]
hello what did you want to say?

**Comment by Quentin-M58** [comment]
Hi, I see the problem, for me, is fixed.
